### ISSUE TITLE
Dialog not launching correctly

### ISSUE THREAD

**Issue by lgharr** [swiftDialog/swiftDialog#258]
Every once in a while when I run a dialog, i get the following error in my output
```shell
/usr/local/bin/dialog: line 29:  1653 Illegal instruction: 4  /usr/bin/sudo -u $currentUser "$dialogbin" "$@"
``` 
I don't have $currentUser as a variable in my script, and line 29 of my script is a comment. The script continues to run, but the dialog window does not get created. my dialog prompt is 
```shell
$dialog --title "Welcome to $clientName" --message "Please wait, you will be prompted to restart your device when complete." --blurscreen --button1disabled --progresstext --progress 7 --bannerimage "$clientBanner" --alignment center &

``` 
 and $dialog calls 
```shell
dialog="/usr/local/bin/dialog
``` 

I am running my full script as sudo, while on the machine in question and not remotely. Possible pattern: This time it happened after a new install of dialog, where as it consistently works when dialog is already installed.

**Comment by bartreardon** [comment]
also, when creating an issue please use one of the issue templates and include all the relevant details for replicating the issue.

**Comment by bartreardon** [comment]
not sure if this is an issue but if it is, should be fixed in 2.3
